content/english/chatbot/activity-1.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Activity 1: Meet ALICE"
+draft: false
+weight: 4
+---
+
+## Meet ALICE
+Before we start building our own chatbot, let's meet ALICE and ask it some questions.
+
+<a class="my-2 mx-4 btn btn-info" href="https://www.pandorabots.com/pandora/talk?botid=b8d616e35e36e881" target="_blank">Try ALICE</a>

```

Comment:
Can we mention that you should continue to have this webpage open and available?